crates/bevy_window/src/window.rs

```diff
@@ -776,6 +780,26 @@ impl Default for CursorOptions {
     }
 }
 
+
+/// Icon data for a [`Window`].
+#[derive(Component, Debug, Clone, Default)]
+#[cfg_attr(
+    feature = "bevy_reflect",
+    derive(Reflect),
+    reflect(Component, Debug, Default, Clone)
+)]
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
We should think about whether we want to also toss in `serde` derives. I would probably omit them, but I know @ChristopherBiscardi has opinions.